configure.ac

```diff
@@ -188,6 +188,11 @@ AC_ARG_ENABLE(rdpsndaudin, AS_HELP_STRING([--enable-rdpsndaudin],
               [], [enable_rdpsndaudin=no])
 AM_CONDITIONAL(XRDP_RDPSNDAUDIN, [test x$enable_rdpsndaudin = xyes])
 
+AC_ARG_ENABLE(openrc, AS_HELP_STRING([--enable-openrc],
+              [Use openrc in (default: no)]),
+              [], [enable_openrc=no])
+AM_CONDITIONAL(XRDP_OPENRC, [test x$enable_openrc = xyes])
```

Comment:
Is it possible to autodetect OpenRC like HAVE_SYSTEMD?

Reply:
Unfortunately, they also use etc/init.d to store their configuration files, so that rules out the easiest way.   I checked my running system, and it doesn't have any systemd binaries installed, so I suppose a valid check might be just to check for systemd.  The gentoo wiki warns that systemd can show up even if you don't use it due to package dependencies.  So not 100% sure that is reliable.  I know for their packages they don't autodetect, you specify in your "use flags" whether you want systemd or not.

Reply:
After poking around the gentoo forums, seems like trying to autodetect this is discouraged.  For gentoo, you specify when you install a package whether you want to use systemd or not, so they state you should have an option to tell it whether to use systemd or not.